integration-tests/manifests/cloudwatch-agent-daemonset.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: cloudwatch.aws.amazon.com/v1alpha1
+kind: AmazonCloudWatchAgent
+metadata:
+  name: cloudwatch-agent
+  namespace: amazon-cloudwatch
+spec:
+  mode: daemonset
+  serviceAccount: cloudwatch-agent
+  image: public.ecr.aws/cloudwatch-agent/cloudwatch-agent:1.300031.1b317
```

Comment:
should we use the latest cwagent? might want to ensure we are compatible with the latest released agent

`public.ecr.aws/cloudwatch-agent/cloudwatch-agent:latest`

Reply:
I will fix.